composer.json

```diff
@@ -14,6 +14,9 @@
         "enum",
         "map"
     ],
+    "require": {
+        "php": ">=7.1 <8.3"
```

Comment:
Is there a reason for the upper limit limitation? That'd require us to constantly up this with every PHP release.